sample/src/main/res/layout/activity_main.xml

```diff
@@ -9,6 +9,25 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:text="@string/hello_world"/>
+  <TextView
+      android:id="@+id/repeated_view_1"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:visibility="gone"
```

Comment:
Totally... the thing is that I don't know if it's common to check the context in the "given" part instead of setting it. But... let's give it a try :·)